src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -213,4 +229,10 @@ private void deleteReplay() {
       onDeleteListener.run();
     }
   }
+
+  private void applyWatchedReplayHighlight() {
+    replayTileRoot.setStyle("-fx-background-color: -card-watched-color;");
+    replayTileRoot.applyCss();
+  }
```

Comment:
Rather than applying the style directly a css pseudo class name should be used instead.